src/BladeFontAwesomeServiceProvider.php

```diff
@@ -64,8 +64,12 @@ private function registerProIcons(Factory $factory, string $proIconsPath, Reposi
         $factory->add('fontawesome-duotone', array_merge(['path' => "{$proIconsPath}/duotone"], $config->get('blade-fontawesome.duotone', [])));
         $factory->add('fontawesome-thin', array_merge(['path' => "{$proIconsPath}/thin"], $config->get('blade-fontawesome.thin', [])));
 
-        if (is_dir("{$proIconsPath}/sharp")) {
-            $factory->add('fontawesome-sharp', array_merge(['path' => "{$proIconsPath}/sharp"], $config->get('blade-fontawesome.sharp', [])));
+        $factory->add('fontawesome-sharp-light', array_merge(['path' => "{$proIconsPath}/sharp-light"], $config->get('blade-fontawesome.sharp-light', [])));
+        $factory->add('fontawesome-sharp-regular', array_merge(['path' => "{$proIconsPath}/sharp-regular"], $config->get('blade-fontawesome.sharp-regular', [])));
+        $factory->add('fontawesome-sharp-solid', array_merge(['path' => "{$proIconsPath}/sharp-solid"], $config->get('blade-fontawesome.sharp-solid', [])));
+
+        if (is_dir("{$proIconsPath}/custom")) {
```

Comment:
Should the paths here be `custom-icons`?

https://fontawesome.com/v6/docs/desktop/setup/kit-download#custom-subsetted-kits

Or is this something different?

Reply:
they are exactly for the custom icons of the fontawesome kits, for example I generated 3 with the wizard icons

[Image](https://prnt.sc/FPjwM_8PqRfo)

Reply:
Nice, my bad I misread the code. 😬 Looks great!